crates/router/src/configs/settings.rs

```diff
@@ -217,11 +219,13 @@ pub struct Secrets {
     pub jwt_secret: String,
     #[cfg(not(feature = "kms"))]
     pub admin_api_key: String,
-
+    pub master_enc_key: String,
```

Comment:
If you'd prefer to provide a default value only for the purpose of local development, you can add an obvious placeholder value in the `config/development.toml` file.